maliput_ros/test/maliput_ros/ros/maliput_query_node_test.cc

```diff
@@ -286,9 +288,15 @@ TEST_F(MaliputQueryNodeAfterConfigurationTest, ConfigureStateAdvertisesServices)
 
   ASSERT_STREQ(service_names_and_types[kRoadGeometryServiceName][0].c_str(), kRoadGeometryServiceType);
   ASSERT_STREQ(service_names_and_types[kJunctionServiceName][0].c_str(), kJunctionServiceType);
+  ASSERT_STREQ(service_names_and_types[kLaneServiceName][0].c_str(), kLaneServiceType);
   ASSERT_STREQ(service_names_and_types[kSegmentServiceName][0].c_str(), kSegmentServiceType);
 }
 
+template <typename T>
+struct ServiceType {
+  using type = T;
+};
+
```

Comment:
Are you using it?

Reply:
Removed in a future PR. Sorry about that!